lib/noticed/delivery_methods/email.rb

```diff
@@ -50,6 +50,20 @@ def format
         end
         params.merge(recipient: recipient, record: record)
       end
+
+      def args
```

Comment:
I'm not sure if we should standardize on `args` or `arguments`? I usually don't like to abbreviate, but Its kind of like`params` and `parameters`?